app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#008577</color>
     <color name="colorPrimaryDark">#00574B</color>
     <color name="colorAccent">#D81B60</color>
+    <color name="white">#FFFFFF</color>
```

Comment:
Can we use this color resource for the `android:fill` colors of `ic_flag` and `ic_map`?

Reply:
Not necessary as ic_flag and ic_map are drawable assets